tools/build-release-branch.sh

```diff
@@ -81,6 +81,26 @@ function modify_svnignore {
 	git commit .svnignore -m "Updated .svnignore"
 }
 
+# Get the latest version of all our packages
+# And use it in our composer.json files in the shipped version of the plugin.
+function update_package_versions {
+	echo "Updating package versions..."
+
+	# Prefer stable versions in release branches.
+	sed -i '' -e '/minimum-stability/ s/dev/stable/' composer.json
+
+	# Switch Automattic packages to a stable version.
+	for line in $( cat composer.json 2>/dev/null ); do
+		if [[ $line == *"automattic/jetpack-"* ]]; then
+			echo 'found one.'
+			PACKAGE=$( echo $line | cut -d'"' -f 2 )
+			composer require $PACKAGE
```

Comment:
We can do `composer require PACKAGE1 PACKAGE2 etc` too which may save a little time.